public/locales/en/common.json

```diff
@@ -173,5 +173,7 @@
   "signOut": "Sign Out",
   "availableBalance": "Available Balance",
   "openPositions": "Open Positions",
-  "reset": "Reset"
+  "reset": "Reset",
+  "connectWalletBuy": "Connect Wallet + Buy",
```

Comment:
we can not do `+` actions with torus
so if the wallet is not connected it should be `Connect wallet to continue`

Reply:
can we do this with another if clause instead `(if torus){...do things}`? i would put three options instead- as the current design shows this updated text.

@shekhar-shubhendu

Reply:
i also dont think the + text necessarily implies that the action will occur immediately but maybe @anitasengupta would feel differently

Reply:
I do not want to have if(torus) before every action as we might need to integrate with other providers in the future.
wallet integration should not be tightly coupled.

Reply:
this is changed